rqd/rqd/rqmachine.py

```diff
@@ -369,11 +369,9 @@ def getPathEnv(self):
 
     @rqutil.Memoize
     def getTempPath(self):
-        """Returns the correct mcp path for the given machine"""
+        """Returns the correct temp path for the given machine"""
         if platform.system() == "win32":
             return win32api.GetTempPath()
-        elif os.path.isdir("/mcp/"):
```

Comment:
If we allow for customizable temp dirs, we'll want to make sure it gets added back in here.